contracts/tsconfig.json

```
@@ -1,5 +1,10 @@
 {
-  "include": ["src/**/*.ts"],
+  "include": [
+    "src/access/witnesses/**/*.ts",
+    "src/security/witnesses/**/*.ts",
+    "src/token/witnesses/**/*.ts",
+    "src/utils/witnesses/**/*.ts",
+  ],
```

What do you think of this pattern?
```suggestion
  "include": [
    "src/**/witnesses/**/*.ts"
  ],
  "exclude": ["src/archive/"]
```